jni_gen/test/simple_package_test/java/dev/dart/simple_package/Example.java

```diff
@@ -0,0 +1,55 @@
+package dev.dart.simple_package;
+
+public class Example {
+  // static final fields
```

Comment:
Maybe remove these comments.

Unless you'd like to signify why these language constructs need to stay. (E.g. Static fields for testing JNI bindings to static fields.) But I'd say it is obvious from the context that this a test file that should not be modified to remove the unused things. (We have checked in the expected generated bindings, the test would start failing anyway.)